frontend/src/scenes/settings/organization/InviteModal.tsx

```diff
@@ -97,18 +112,30 @@ export function InviteRow({ index, isDeletable }: { index: number; isDeletable:
                         }}
                     />
                 )}
-                {isDeletable && (
-                    <LemonButton icon={<IconTrash />} status="danger" onClick={() => deleteInviteAtIndex(index)} />
-                )}
             </div>
+            <div className="flex-1 flex gap-1 items-center justify-between">
+                <LemonSelect
```

Comment:
What happens if a member is inviting someone? Should we just hide this?

Reply:
Will hide if there is only one allowed option.